google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/stub/BigtableChannelPrimer.java

```diff
@@ -106,6 +132,11 @@ private void sendPrimeRequests(ManagedChannel managedChannel) throws IOException
           e = e.getCause();
         }
         LOG.warning(String.format("Failed to prime channel: %s", e));
+        if (responseMetadata.getMetadata() != null) {
+          LOG.warning(
+              "Encrypted debug header is "
+                  + responseMetadata.getMetadata().get(ENCRYPTED_HEADERS_KEY));
```

Comment:
I think this needs to be more user friendly. "Channel priming failed, which implies a connection issue. When opening a support case, include this encrypted debug values in the support case"